patches/components-omnibox-browser-BUILD.gn.patch

```diff
@@ -10,3 +10,11 @@ index 44e8cc4556303b463d424170af2f93bed0b9f8d4..fdd6fcc8a69d9dd487849076892e21f6
  
    configs += [ "//build/config/compiler:wexit_time_destructors" ]
  
+@@ -929,6 +930,7 @@ fuzzer_test("search_suggestion_parser_fuzzer") {
+ 
+ mojom("mojo_bindings") {
+   sources = [ "omnibox.mojom" ]
++  generate_legacy_js_bindings = true
```

Comment:
Without this, the page only generates the WebUI bindings, which we can't use from out JS files (yet). We should probably look into this and work out what we need to do.

Reply:
Oh yeah, I went fairly deep down this rabbit hole when I was initially trying to get this working. Unfortunately, setting the 'webui_module_path` stops the normal JS/TS generation from working.

There are several things about the `mojom-webui.ts` files which are a bit different, but the main thing is that they're expecting to be used from Chromium's Rollup/TsConfig/GN process.

We probably should work out how to make our build process support these mojom webui files, but even when I did get them loading on the NTP they weren't working properly (because of some transitive deps which either they bundle and we don't, or we bundle and they don't).

In the end, this was the easiest solution, as it generates both the `mojom-webui.ts` files and the `mojom.m.js` file that we want.

Reply:
Yeah this seems to be the easiest way 👍